### ISSUE TITLE
Improve  prominence and curation of drug inventory

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2517]
We need to make the drug inventory more visible 
https://github.com/pombase/website/issues/1408
https://www.pombase.org/browse-curation/drugs-known-pombe-targets

We should *definitely*  flag these inhibitors on the pages for the genes.... 
At present, the table  munges  orthogonal data into column 2, and we *hate* that don't we (my bad).

Stage 1
split
Targeted gene product, process, etc.
into 2 columns :

gene, process, note?

 We can then make a table for import to display on the relevant gene pages.

Stage 2.
Display gene-specific drug targets in a new gene page section

**Comment by ValWood** [comment]
we can "news item" and tweet this with requests for input

**Comment by ValWood** [comment]
@Antonialock  is there a clever way to expand this list?

**Comment by Antonialock** [comment]
I think drugbank and chembl do protein targets

I have requested a download account from drugbank. I think we can access chembl without login, just need to do some digging

so if we cross reference the orthologs we should be able to find what proteins the drug targets in human
(we could have "verified in pombe" and "predicted from orthology" tags?)

**Comment by Antonialock** [comment]
keep in mind many drugs are broad spectrum...

**Comment by Antonialock** [comment]
(i.e. they might target protein X Y AND Z (and more...) 

often not all targets are known

**Comment by ValWood** [comment]
yep. I think we can do a better job when a specific target is known though

**Comment by Antonialock** [comment]
(especially for protein kinase inhibitors - at least that's my understanding)

**Comment by Antonialock** [comment]
well if it is a curated "drug target" entry on a gene page we could always include a count column saying how many proteins are known to a target of the drug

I have really not thought about this, but we could also tag with BPs....e.g cytotoxics ( topo II inhibitors etc)

**Comment by ValWood** [comment]
Maybe you should maintain this table? it fits in with your Healx work?

We should initially just reorganize the table, and publicize it. We can integrate it better later.

**Comment by mah11** [comment]
What is the task for me? If it's to add or change anything on the website, it belongs on the website tracker (although most of the comment above are about curating the content, and aimed at Antonia, so I'll add her to the assignees).

If there is a website task, could you please make it clear on the website ticket exactly what you want done? I don't know what would be "more prominent" to you or to users, so need suggestions.

**Comment by mah11** [comment]
oh, and this:

> Stage 2.
> Display gene-specific drug targets in a new gene page section

is a different website task for Kim (probably worth its own ticket)

**Comment by ValWood** [comment]
I would prefer it to be available from a menu once it is extended. I can never find it....

**Comment by ValWood** [comment]
This is  a placeholder/brainstorming ticket until we think how to do it.

I assigned to Antonia only. When we decide exactly what to do we will probably close this and open new tickets.

**Comment by ValWood** [comment]
For this ticket, just clean up the table  so that it  has separate columns for  gene, processs, and add a note.

Then we can see clearly which have identified specific targets, and maintain properly.

I'll open new tickets once this part is done

**Comment by ValWood** [comment]
Here  can you just split out the orthogonal data into separate table columns, then we can close this. We won't do any work to expand it but can  make a news item and ask for additions.

**Comment by Antonialock** [comment]
tidied up I think
